src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipOptionsType.php

```diff
@@ -49,15 +54,58 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TranslatableType::class,
                 [
                     'type' => TextType::class,
+                    'label' => $this->trans('Delivery prefix', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans('Prefix used for delivery slips.', 'Admin.Orderscustomers.Help'),
+                    'multistore_configuration_key' => 'PS_DELIVERY_PREFIX',
+                    'options' => [
+                        'constraints' => [
+                            new Regex([
+                                'pattern' => '[\<]',
+                                'match' => false,
+                                'message' => $this->trans(
+                                    'The "<" character is forbidden.',
```

Comment:
```suggestion
                                    'The %s character is invalid.',
```

Invalid characters are not indicated beforehand. This is not a good practice as the user could have to try multiple times before finally managing to save their changes. Could we add a help text with the list of invalid characters under the field? 

We can use the following existing string: 'Invalid characters:' on Admin.Notifications.Info